iphone/Classes/TiUITextWidget.m

```diff
@@ -202,6 +202,7 @@ - (void)setAutocapitalization_:(id)value
 - (void)textWidget:(UIView<UITextInputTraits> *)tw didFocusWithText:(NSString *)value
 {
   TiUITextWidgetProxy *ourProxy = (TiUITextWidgetProxy *)[self proxy];
+  [[ourProxy keyboardAccessoryView] setBounds:CGRectMake(0, 0, 0, [ourProxy keyboardAccessoryHeight])];
```

Comment:
Please do not set the frame manually if possible. We have a layout helper to fill a height - although I am missing the name right now. It may be *okay* here, because the keyboard always fills the whole width and has a static position, but I feel this may produce issues with splitted keyboards on the iPad landscape.

Reply:
I think you are talking about method "[TiUtils setView:view positionRect:view.bounds]". But this is giving problem in iOS 10 and below. So there is no problem in using this.